ci-operator/step-registry/rosa/sts/cluster/provision/rosa-sts-cluster-provision-commands.sh

```diff
@@ -99,6 +99,11 @@ fi
 echo "Choosing openshift version ${OPENSHIFT_VERSION}"
 
 # Switches
+EC2_METADATA_HTTP_TOKENS_SWITCH=""
+if [[ -n "${EC2_METADATA_HTTP_TOKENS}" && "$HOSTED_CP" != "true" ]]; then
```

Comment:
I think you can just check the option is set or not. In case in future it supports hosted cp

Reply:
yes, but current `EC2_METADATA_HTTP_TOKENS`'s default is `optional`
https://github.com/openshift/release/blob/15d4bf02524dd76685e4387f378adeb1e69ddff9/ci-operator/step-registry/rosa/sts/cluster/provision/rosa-sts-cluster-provision-ref.yaml#L34

Reply:
So I chose to fix it this way. If future hosted CP supports it, we just need to remove this change.